PSI Food waste/Pages/Shared/_Layout.cshtml

```diff
@@ -21,6 +21,9 @@
                         <li class="nav-item">
                             <a class="nav-link text-dark" asp-area="" asp-page="/Index">Home</a>
                         </li>
+                        <li class="nav-item">
+                            <a class="nav-link text-dark" asp-area="" asp-page="/AddRestaurant">Add Restaurant </a>
```

Comment:
You forgot to add /Forms at asp-page, the button currently doesn't work. It should look like asp-page="/Forms/AddRestaurant".